src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -1759,7 +1761,7 @@ resource pipeline_ToIngestion 'Microsoft.DataFactory/factories/pipelines@2018-06
         typeProperties: {
           variableName: 'destinationFolder'
           value: {
-            value: '@replace(concat(variables(\'metric\'), \'/\',variables(\'date\'),\'/\',variables(\'scope\')),\'//\',\'/\')'
+            value: '@replace(concat(variables(\'scope\'), \'/\',variables(\'date\'),\'/\',variables(\'metric\')),\'//\',\'/\')'
```

Comment:
If we make this change, we need to include a script to rearrange the existing content. There was a lot of pushback from forcing a re-export in 0.2 and now that exports limits to 13mo, that would result in lost data for people who want to maintain more. I'd like to avoid that going forward. We can handle the upgrade in a separate PR.

Can you also update data-processing.md to align with whatever we do? That can be in a separate PR.

Reply:
Reverted